c-layer/lib.cxx

```diff
@@ -152,10 +181,17 @@ void resolvable_select(const char* name, enum RESOLVABLE_KIND kind, struct Statu
 
   status->state = Status::STATE_SUCCEED;
   status->error = NULL;
-  selectable->setToInstall(zypp::ResStatus::APPL_HIGH);
+  auto value = convert_selected(who);
```

Comment:
Bad naming here... `auto`, `value` and `convert` do not tell me what the result is... yes it's an internal helper but it should be more informative: `auto value = transactby_from(who);`